### ISSUE TITLE
load-bash-aliases-into-eshell - minor bug

### ISSUE THREAD

**Issue by j3kyl** [daviderestivo/emacs-config#2]
Hey, there are some minor incomplete name to the element  : 

```
(defun lbsa/bash-to-eshell-aliases (BASHFILE)
  "Takes a BASHFILE, trims it to a list of alias commands, and inserts them as eshell aliases."
  (interactive "f")
  (eshell)
  (dolist (element (lbsa/extract-bash-aliases (lbsa/read-lines-from-file BASHFILE)))
    (let ((trimmed (replace-regexp-in-string "=" " " element)))
      (goto-char (point-max))
      (insert trimmed)
      (eshell-send-input))))
```

Thats all!

Care to make it a standalone package?
Thanks!

**Comment by daviderestivo** [comment]
Fixed [89b71e3](https://github.com/daviderestivo/emacs-config/commit/89b71e32a4f4cfa8c2d13c96f01e3c776e458cb6).

I was planning to make a package and put it on Melpa but I still need to work on it ...

**Comment by daviderestivo** [comment]
Hi,

I have just releases this small utility library as a standalone package. If you're interested you can get it from here: https://github.com/daviderestivo/load-bash-shell-aliases

P.S.: I've done some code cleanup and small improvements.

Regards,
Davide
